tools/dataMigration/fraSpecs.ts

```diff
@@ -5262,8 +5262,9 @@ export const FraSpecs: Record<string, SectionSpec> = {
                 variableName: 'other_or_unknown',
                 migration: {
                   colNames: ['1990', '2000', '2010', '2015'],
-                  calcFormula:
-                    'extentOfForest.forestArea - (forestOwnership.private_ownership || 0) - (forestOwnership.public_ownership || 0)',
+                  calcFormula: `forestOwnership.private_ownership && forestOwnership.public_ownership 
```

Comment:
- || 0 is no more necessary
- in this case backtick is ok but not necessary, we can keep it

'forestOwnership.private_ownership && forestOwnership.public_ownership 
    ? extentOfForest.forestArea - forestOwnership.private_ownership - forestOwnership.public_ownership 
    : null'

Reply:
yes, because one of the two can still be null, it's safer that way

Reply:
backtick is used to put the formula in multiple lines, for readability